### PR TITLE
Added addMultipleFromRequest and addAllFromRequest in medialibrary

### PR DESCRIPTION
**Pull request by AlexVanderbist**
Docs for spatie/laravel-medialibrary#551

### CODE REVIEW

File: resources/views/laravel-medialibrary/v5/api/adding-files.md [hunk 1]
```diff
@@ -52,15 +52,56 @@ public function addMediaFromUrl(string $url)
...
+/**
+ * Add multiple files from the current request to the medialibrary.
+ * 
+ * @param string[] $keys
+ *
+ * @return \Spatie\MediaLibrary\FileAdder\FileAdder[]
+ *
+ * @throws \Spatie\MediaLibrary\Exceptions\FileCannotBeAdded
+ */
+public function addMultipleMediaFromRequest(array $keyNames): array
```
Comment (freekmurze) [inline resources/views/laravel-medialibrary/v5/api/adding-files.md]: I refactored this to a collection in the code 😄

**Review by freekmurze — CHANGES_REQUESTED** [review]

File: resources/views/laravel-medialibrary/v5/api/adding-files.md [hunk 2]
```diff
@@ -52,15 +52,56 @@ public function addMediaFromUrl(string $url)
...
+
+```php
+/**
+ * Add all files from the current request to the medialibrary.
+ *
+ * @return \Spatie\MediaLibrary\FileAdder\FileAdder[]
+ *
+ * @throws \Spatie\MediaLibrary\Exceptions\FileCannotBeAdded
+ */
+public function addAllMediaFromRequest(): array
```
Comment (freekmurze) [inline resources/views/laravel-medialibrary/v5/api/adding-files.md]: Actually returns a collection now

File: resources/views/laravel-medialibrary/v5/api/adding-files.md (hunk 1 above)
Comment (AlexVanderbist) [reply resources/views/laravel-medialibrary/v5/api/adding-files.md]: I was too fast :(

**Review by freekmurze — APPROVED** [review]

**Comment by freekmurze** [comment]
Much better, thanks!

**Pull request merged**